CreatubblesAPIClientIntegrationTests/Spec/SearchTag/SearchTagsFetchResponseHandlerSpec.swift

```diff
@@ -64,7 +65,10 @@ class SearchTagsFetchResponseHandlerSpec: QuickSpec
                 {
                     done in
                     sender.logout()
-                        sender.send(request, withResponseHandler: SearchTagsFetchResponseHandler
+                    sender.clearCredentials()
```

Comment:
Why it cannot be a part of `logout` method ?

Reply:
I guess it's a question to @mmiedlarz, why public credentials aren't deleted here when logging out.